waspc/examples/todoApp/src/client/App.jsx

```diff
@@ -0,0 +1,25 @@
+import logout from "@wasp/auth/logout.js";
```

Comment:
Having a root component opened up a place for `Main.css` import and having a `Logout` button present in the header of each page.